sysconfig/cloudinit.go

```diff
@@ -317,6 +324,8 @@ func RestrictCloudInit(state CloudInitState, opts *CloudInitRestrictOptions) (Cl
 		fallthrough
 	case CloudInitUntriggered:
```

Comment:
(nitpick^2) this could be slightly more compact: `case CloudInitUntriggered, CloudInitNotFound:`